mmv1/third_party/terraform/tests/resource_compute_region_instance_group_manager_test.go.erb

```diff
@@ -1295,15 +1295,15 @@ resource "google_compute_region_instance_group_manager" "igm-rolling-update-poli
   distribution_policy_zones = ["us-central1-a", "us-central1-f"]
   target_size               = 3
   update_policy {
-    type                         = "PROACTIVE"
-    instance_redistribution_type = "NONE"
-    minimal_action               = "REPLACE"
```

Comment:
Looks like minimal_action was removed here, but minimal_action is a required field for this resource, so the test cannot pass

Reply:
I made this field optional with the default value REPLACE. This is the behavior of the REST api. minimal_action is not required but if it is not specified it will be set to REPLACE. I was able to execute tests this time using my environment and those tests passed there.

Reply:
Why are we removing the Required parameter on this field? I believe we intentionally made this required to force users to specify REPLACE explicitly. Defaulting to REPLACE has somewhat dangerous implications so we try to make users make the choice themselves

Reply:
I reverted this change. Lets decouple this discussion from this PR.